src/amo/components/Addon/index.js

```diff
@@ -280,7 +280,9 @@ export class AddonBase extends React.Component {
     let errorBanner = null;
     if (errorHandler.hasError()) {
       log.error('Captured API Error:', errorHandler.capturedError);
```

Comment:
While you're in here, could you make this `log.warn` instead of `log.error`? That was a mistake that I'm only noticing right now. When we use `log.error` it shows up in our error log as something we need to fix. This is not something that needs fixing; it is an expected outcome of the user's action.